src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -50,8 +50,9 @@ static NodeFunctionInvoker()
             Initialize();
         }
 
-        internal NodeFunctionInvoker(ScriptHost host, BindingMetadata trigger, FunctionMetadata functionMetadata, Collection<FunctionBinding> inputBindings, Collection<FunctionBinding> outputBindings)
```

Comment:
Good point. I focused on the compilation/.NET invoker, but I'll update those as part of this PR.